software/contrib/polyrhythmic_sequencer.py

```diff
@@ -55,11 +55,66 @@
 VOLT_PER_OCT = 1 / 12
 
 NOTES = [
-    "C0", "C#0", "D0", "D#0", "E0", "F0", "F#0", "G0", "G#0", "A0", "A#0", "B0",
-    "C1", "C#1", "D1", "D#1", "E1", "F1", "F#1", "G1", "G#1", "A1", "A#1", "B1",
-    "C2", "C#2", "D2", "D#2", "E2", "F2", "F#2", "G2", "G#2", "A2", "A#2", "B2",
-    "C3", "C#3", "D3", "D#3", "E3", "F3", "F#3", "G3", "G#3", "A3", "A#3", "B3",
-    "C4", "C#4", "D4", "D#4", "E4", "F4", "F#4", "G4", "G#4", "A4", "A#4", "B4",
+    "C0",
```

Comment:
Is it possible to configure Black to ignore certain formatting rules, or allow us to add a comment to ignore and not format the preceding line or block? This formatting change doesn't add value in my opinion, and removes context the previous formatting gave.

Reply:
Yes! We could use `# fmt: off` and `# fmt: on` to ignore blocks. I'll go ahead and revert this instance already.